package.json

```diff
@@ -53,7 +54,16 @@
 					"description": "Path to the buf binary"
 				}
 			}
-		}
+		},
+		"languages": [
```

Comment:
Does this do it just for the development of this plugin, or does this do it for users who install this plugin?

Reply:
I tested this in the same way I tested the plugin during development, and it worked the way I wanted, so I am confident this is going to make this file association for any users using this extension.